controllers/topology/util.go

```diff
@@ -114,18 +114,31 @@ func (l *topologyReconcileLogger) Into(ctx context.Context) (context.Context, lo
 	return ctrl.LoggerInto(ctx, l.Logger), l
 }
 
-// KRef return a reference to a Kubernetes object in the same format used by kubectl commands (kind/name).
-type KRef struct {
+// KObj return a reference to a Kubernetes object in the same format used by kubectl commands (kind/name).
+// Note: We're intentionally not using klog.KObj as we want the kind/name format instead of namespace/name.
+type KObj struct {
 	Obj client.Object
 }
 
-func (ref KRef) String() string {
+func (ref KObj) String() string {
 	if ref.Obj == nil {
 		return ""
 	}
 	return fmt.Sprintf("%s/%s", ref.Obj.GetObjectKind().GroupVersionKind().Kind, ref.Obj.GetName())
 }
 
+// KRef return a reference to a Kubernetes object in the same format used by kubectl commands (kind/name).
+type KRef struct {
```

Comment:
Same as above, could we have an internal package instead of making this an API?

Reply:
Done